sdk/iot/core/inc/az_iot_sas_token.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef AZ_IOT_SAS_TOKEN_H
```

Comment:
we hide the internal identifiers that bring no benefits to the user behind underscores, so intellisense would put them at the back of the list. However, identifiers that start with an underscore and a capital letter are reserved. So we name uppercase internal identifiers like this: `_az_IOT_SAS_TOKEN_H`.

Reply:
Fixed.